### PR TITLE
[FIX] synchronize.py: Delete tmp folder into /app/data/vcs

### PR DESCRIPTION
**Pull request by JesusZapata**
Delete all tmp folder into /app/data/vcs

### CODE REVIEW

File: synchronize.py [hunk 1]
```diff
@@ -194,6 +181,20 @@ def sync(self):
                                'branches (id=%s, name=%s)', repo['id'],
                                repo['name'])
             self._wlapi.import_from_runbot(repo, branches)
+        cmd = []
+        if configuration.has_section('docker'):
+            weblate_container = configuration.get('docker', 'name')
+        if weblate_container:
```
Comment (moylop260) [inline synchronize.py:187]: `Undefined variable` (if the condition below is not acomplished)

File: synchronize.py [hunk 2]
```diff
@@ -194,6 +181,20 @@ def sync(self):
...
+            cmd.extend(['docker', 'exec', weblate_container])
+        cmd.extend(['find', '/app/data/vcs',
```
Comment (moylop260) [inline synchronize.py:189]: Use this command where you are calling `sync` method using.

```python
try:
    sync()
finally:
    clean()

File: synchronize.py (hunk 1 above)
Comment (JesusZapata) [reply synchronize.py:187]: @moylop260 
This PR fixes this comment

File: synchronize.py (hunk 2 above)
Comment (JesusZapata) [reply synchronize.py:189]: @moylop260 
This PR fixes this comment

**Pull request merged**